examples/cfme.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: oVirt CFME deployment
+  hosts: localhost
+  connection: local
+  gather_facts: false
+
+  vars_files:
+    # Contains encrypted `engine_password` and `metrics_password`
+    # varibale using ansible-vault
+    - passwords.yml
+
+  vars:
+    engine_fqdn: ovirt-engine.example.com
+    engine_user: admin@internal
+    engine_password: 123456
+
+    miq_vm_name: cfme_59
+    miq_qcow_url: https://cdn.example.com/cfme-rhevm-5.9.1.2-1.x86_64.qcow2
+    miq_vm_cluster: mycluster
+    miq_vm_root_password: securepassword
+    miq_vm_cloud_init:
+      host_name: "{{ miq_vm_name }}"
```

Comment:
Is ovirt-guest-agent-common installed on downstream appliance, or do we want to install it? (Is the appliance registered to RHSM even?)